src/Resources/config/serializer/Symfony.Component.Debug.Exception.FlattenException.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <serializer>
-    <class name="Symfony\Component\Messenger\Stamp\RedeliveryStamp" exclusion-policy="ALL">
+    <class name="Symfony\Component\Debug\Exception\FlattenException" exclusion-policy="ALL">
```

Comment:
can you add debug as a dev dependency please

Reply:
> can you add debug as a dev dependency please

Symfony/Debug is already a dev dependency.